### ISSUE TITLE
Chromosomal rearrangement

### ISSUE THREAD

**Issue by evanc1648** [obophenotype/human-phenotype-ontology#10891]
**Preferred term label:**
Chromosomal rearrangement

**Synonyms**
Chromosomal rearrangements
Structural chromosomal rearrangement
Structural chromosomal rearrangements

**Definition (free text, please give PubMed ID)**
A change in the structure of one or more chromosomes
(PMID: 35701783 and NCIT:C36541)

**Parent term (use hpo.jax.org/app)**
HP:0031411

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
DiGeorge Syndrome (OMIM:188400)
Williams syndrome (ORPHA:904)
Williams-Beuren syndrome (OMIM:194050)
Prader-Willi syndrome (OMIM:176270)
Prader-Willi syndrome (ORPHA:739)
More in PMID: 35701783

**Your nano-attribution (ORCID)**
0009-0001-4998-8617

**Comment by pnrobinson** [comment]
@evanc1648 
We would consider this to be a genotype rather than a phenotype, so it is out of scope for HPO.
It is currently a little difficult to represent chromosomal anomalies in detail in phenopackets, but in principle VRS and/or ISBN allows us to, and so it would be good to look into this more if you are interested in chromosomal analysis, but that would not be represented by HPO terms.
